src/plugins/visualizations/server/embeddable/visualize_embeddable_factory.ts

```diff
@@ -86,6 +94,9 @@ export const visualizeEmbeddableFactory = (): EmbeddableRegistryDefinition => {
           byValueMigrateTagcloud,
           byValueAddDropLastBucketIntoTSVBModel
         )(state),
+      '7.15.0': (state: SerializableRecord) => state,
```

Comment:
This and the next line doesn't seem to do anything, are those leftovers?

Reply:
Removed lines with unneeded versions.